### PR TITLE
[SET-403] add default value and var if specified to run as build script

### PR DESCRIPTION
**Pull request by baranowb**
Issue: https://issues.redhat.com/browse/SET-403

### CODE REVIEW

File: cryo.sh [hunk 1]
```diff
@@ -69,7 +69,7 @@ fi
 
 # use HarmoniaOperationCenter which requires env.ENV_HARMONIA_BUILD_SH pointing to eap-job.sh
 HARMONIA_HOME="$(cd "${WORKSPACE}/../harmonia" && pwd)"
-export ENV_HARMONIA_BUILD_SH=${HARMONIA_HOME}/eap-job.sh
+export ENV_HARMONIA_BUILD_SH=${HARMONIA_HOME}/${CRYO_HARMONIA_SH:eap-job.sh}
```
Comment (rpelisse) [inline cryo.sh:73]: Just to be safe:
```
${CRYO_HARMONIA_SH:-'eap-job.sh'}
```
So it's clear eap-job.sh is a constant.

**Review by rpelisse — CHANGES_REQUESTED** [review]

File: cryo.sh [hunk 2]
```diff
@@ -5,7 +5,11 @@
 # a git bisect implementation based on github pull requests
 #
 set -eo pipefail
-
+echo "-------------------XXXXXXXXXXX------------------------"
```
Comment (rpelisse) [inline cryo.sh:8]: Can you remove that or add a if [ ${DEBUG} ]; then around? Otherwise, it's going to pollute log output.

**Review by rpelisse — CHANGES_REQUESTED** [review]

File: cryo.sh (hunk 2 above)
Comment (rpelisse) [reply cryo.sh:8]: Or if you keep it:
${@}
${CRYO_HARMONIA_SH}

Comment (baranowb) [reply cryo.sh:8]: Im going to remove it once Im done. There is something fishy going on with values and I dont know yet why soo...

**Pull request closed without merging**